src/buildHTML.js

```diff
@@ -215,14 +258,19 @@ export const makeNullDelimiter = function(options, classes) {
  * function for it. It also handles the interaction of size and style changes
  * between parents and children.
  */
-export const buildGroup = function(group, options, baseOptions) {
+export const buildGroup = function(
+    group: *,
+    options: Options,
+    baseOptions?: Options,
+): HtmlDomNode {
     if (!group) {
         return makeSpan();
     }
 
     if (groupBuilders[group.type]) {
         // Call the groupBuilders function
-        let groupNode = groupBuilders[group.type](group, options);
+        // $FlowFixMe
+        let groupNode: HtmlDomNode = groupBuilders[group.type](group, options);
```

Comment:
`HtmlBuilder<NodeType>` isn't the type union of all possibilities since it's not variant w.r.t. `NodeType` (by virtue of us computing `NodeValue` from it). Really, it should be `{[NodeType]: AnyHtmlBuilder}`. But it's probably not worth the effort. Almost all the typesafety we get is from the definitions `defineFunction` and `definedEnvironment`. Almost nothing else touches `_htmlGroupBuilders`; the couple methods that do expose a very strongly-typed signature.